pkg/storj/metainfo.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storj
+
+import (
+	"context"
+)
+
+// Metainfo represents a database for storing meta-info about objects
+type Metainfo interface {
```

Comment:
As I am looking at the definition of this interface, the name `Store` seems to fit better:
- Store.CreateBucket()
- Store.GetObject()
- Store.ListObjects()
- etc.

Reply:
The problem currently is that they are all in `storj` package, so it would need to be something more specific.

Reply:
Then having the most common `storj.Store` might be fine.